v2v-conversion-host.spec.in

```diff
@@ -4,10 +4,18 @@
 %bcond_with python3
 %endif
 
+%if 0%{?fedora} < 30
```

Comment:
Is this condition correct? Also, is the macro on the following line available in CentOS? If not we need to keep the deps static.

Reply:
It should be available everywhere, but since Fedora 30 it should be enabled by default. I wanted to mark it as such so that it can be removed later. I even tried removing the condition and it didn't help, so I think I'll just keep the requirements hard-coded in the rpm spec.

Reply:
I hard-coded them for now, it can always be made nicer later.